clientimpl.go

```diff
@@ -508,11 +508,13 @@ func (c *userClient) authenticate() *http.Response {
 			resp = c.authenticatev1()
 		}
 	}
-	resp2 := c.HeadAccount(nil)
-	if resp2.StatusCode/100 != 200 {
-		bodyBytes, _ := ioutil.ReadAll(resp2.Body)
-		resp2.Body.Close()
-		return nectarutil.ResponseStub(resp2.StatusCode, "Error response from HEAD on account:\r\n\r\n"+string(bodyBytes))
+	if resp.StatusCode/100 == 200 {
```

Comment:
shouldn't this be: if resp.StatusCode/100 == 2 {  ?

Reply:
Indeed! Thanks